### DISCUSSION TITLE
静态恶意代码逃逸（第十一课）- 汇编语言编写Shellcode加载器 « 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#6] [General]
# 静态恶意代码逃逸（第十一课）- 汇编语言编写Shellcode加载器 « 倾旋的博客

倾旋的博客

http://payloads.online/archivers/2022-02-16/1/

**Comment by j14nj13** [comment]
老板别断更

**Reply by Rvn0xsy** [reply]
尽可能不断更.... 其实这些技术不是新技术，顶多算是一个汇总和记录。

**Comment by fasnow** [comment]
老板太强了，光这几篇都够我吃好久了，刚入门免杀，一头雾水，网上资料千篇一律，质量太低，你这质量也太高了

**Reply by Gs52jrfnxnPXBGAW** [reply]
倾旋师傅 nasm的学习教程有推荐的？

**Reply by Rvn0xsy** [reply]
nasm的资料比较多 ，我觉得最好的就是官方文档了：https://www.nasm.us/xdoc/2.15.05/html/nasmdoc0.html  需要花时间耐心的尝试

**Comment by Gs52jrfnxnPXBGAW** [comment]
倾旋师傅 nasm的学习教程有推荐的？

**Reply by Rvn0xsy** [reply]
nasm的资料比较多 ，我觉得最好的就是官方文档了：https://www.nasm.us/xdoc/2.15.05/html/nasmdoc0.html  需要花时间耐心的尝试

**Comment by MoYiC** [comment]
高质量太爱了

**Comment by b1lack** [comment]
花了一周时间把这系列的看完了。老板太强了。

**Comment by prince-regent** [comment]
断更了吗  是我发现这个宝藏太晚了

**Comment by Rvn0xsy** [comment]
工作原因，很久不写了…

 ------------------原始邮件 ------------------
发件人：prince-regent ***@***.***>
日期：2023-03-20 下午5:07:52
收件人：Rvn0xsy/rvn0xsy.github.io ***@***.***>
抄送：倾旋 ***@***.***>, Comment ***@***.***>
主题：Re: [Rvn0xsy/rvn0xsy.github.io] 静态恶意代码逃逸（第十一课）- 汇编语言编写Shellcode加载器 « 倾旋的博客 (Discussion #6)
断更了吗 是我发现这个宝藏太晚了

**Reply by prince-regent** [reply]
刚开始入门免杀 看到你的文章大受启发,好希望你还能更新

**Comment by Rvn0xsy** [comment]
谢谢，有空会继续更新🦀
 ------------------原始邮件 ------------------
发件人：prince-regent ***@***.***>
日期：2023-03-20 下午9:38:42
收件人：Rvn0xsy/rvn0xsy.github.io ***@***.***>
抄送：倾旋 ***@***.***>, Comment ***@***.***>
主题：Re: [Rvn0xsy/rvn0xsy.github.io] 静态恶意代码逃逸（第十一课）- 汇编语言编写Shellcode加载器 « 倾旋的博客 (Discussion #6)
刚开始入门免杀 看到你的文章大受启发,好希望你还能更新

**Reply by prince-regent** [reply]
小弟等你更新噢

**Comment by cht11** [comment]
膜拜！学习了~
